src/hooks/useValidation/useValidation.js

```diff
@@ -65,20 +66,25 @@ const useValidation = (context = {}) => {
       return {};
     }
 
-    return response.issues.reduce((acc, cur) => {
-      const match = cur.tag.match(/(.{0,3})\[(\d+)\]/);
-      const fieldTag = match[1];
-      const fieldIndex = parseInt(match[2], 10);
-
-      const field = marcRecords.filter(_field => _field.tag === fieldTag)[fieldIndex];
-
-      const existingIssues = acc[field?.id || MISSING_FIELD_ID] || [];
-
-      return {
-        ...acc,
-        [field?.id || MISSING_FIELD_ID]: [...existingIssues, cur],
-      };
-    }, {});
+    // filter out issue 001 for bib records as this field is system generated and expected to be empty
+    return response.issues
+      .filter(issue => !(context.marcType === MARC_TYPES.BIB

```

Comment:
Redundant traversing.
You can achieve the same result just with `.reduce()`:
```
return response.issues.reduce((acc, cur) => {
   if(context.marcType === MARC_TYPES.BIB 
     && [QUICK_MARC_ACTIONS.CREATE, QUICK_MARC_ACTIONS.DERIVE].includes(context.action)
     && cur.tag.startsWith('001')) {
     return acc;
   }
       
   const match = cur.tag.match(/(.{0,3})\[(\d+)\]/);
   const fieldTag = match[1];
   ...
}, {});
```

Reply:
@SKarolFolio - this error only needs to be removed for missing fields, so I moved it outside of this function.